### PR TITLE
Update usage.md

### PR DESCRIPTION
**Pull request by chriswyatt1**
Fixed broken launchpad link. Removed token link, that returned to login

### CODE REVIEW

File: docs/getting-started/usage.md [hunk 1]
```diff
@@ -14,17 +14,17 @@ You can use Tower via either the **online GUI**, using the `-with-tower` option
 
 **1.** Create an account and login into Tower, available free of charge, at [tower.nf](https://tower.nf).
 
-**2.** Create and configure a new [compute environment](../../compute-envs/overview/).
+**2.** Create and configure a new [compute environment](https://help.tower.nf/compute-envs/overview/).
 
-**3.** Start [launching pipelines](../../launch/overview/).
+**3.** Start [launching pipelines](https://help.tower.nf/launch/launchpad/).
```
Comment (pditommaso) [inline docs/getting-started/usage.md:20]: I think it is better to keep the relative paths to make the docs portable. Are the links broken?

File: docs/getting-started/usage.md [hunk 2]
```diff
@@ -14,17 +14,17 @@ You can use Tower via either the **online GUI**, using the `-with-tower` option
...
```
Comment (abhi18av) [inline docs/getting-started/usage.md:19]:
```suggestion
**3.** Start [launching pipelines](../../launch/launchpad/).
```

Comment (abhi18av) [reply docs/getting-started/usage.md:19]: @chriswyatt1 , thanks for the catch on this one. I think that this change ought to be enough to fix it while maintaing the link pattern in the docs.

File: docs/getting-started/usage.md (hunk 1 above)
Comment (pditommaso) [reply docs/getting-started/usage.md:20]: What about the absolute url at line 17? should it stay?

Comment (abhi18av) [reply docs/getting-started/usage.md:20]: Nope, the following should be swapped in for that.

```
**2.** Create and configure a new [compute environment](../../compute-envs/overview/).
```

Comment (pditommaso) [reply docs/getting-started/usage.md:20]: Can you make it please?

File: docs/getting-started/usage.md [hunk 3]
```diff
@@ -14,17 +14,17 @@ You can use Tower via either the **online GUI**, using the `-with-tower` option
 
 **1.** Create an account and login into Tower, available free of charge, at [tower.nf](https://tower.nf).
 
-**2.** Create and configure a new [compute environment](../../compute-envs/overview/).
+**2.** Create and configure a new [compute environment](https://help.tower.nf/compute-envs/overview/).
```
Comment (abhi18av) [inline docs/getting-started/usage.md:17]:
```suggestion
**2.** Create and configure a new [compute environment](../../compute-envs/overview/).
```

File: docs/getting-started/usage.md (hunk 1 above)
Comment (abhi18av) [reply docs/getting-started/usage.md:20]: Sure, I've fixed it with https://github.com/seqeralabs/nf-tower-docs/pull/101/commits/af83a59bf7882feea2d782b38c173469fe4d3a37

Comment (pditommaso) [reply docs/getting-started/usage.md:20]: Cool! How you did before I got a commit proposal. Never understood that

**Review by pditommaso — APPROVED** [review]

**Pull request merged**